### PR TITLE
Added documentation for es cleaner job

### PR DESCRIPTION
**Pull request by cfontes**
Simple documentation for the ES Cleaner Job

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -311,6 +311,19 @@ The self-provision of an Elasticsearch cluster can be disabled by setting the fl
 
 IMPORTANT: At the moment there can be only one Jaeger with self-provisioned Elasticsearch instance per namespace.
 
+=== Elasticsearch index cleaner job
+
+When using `elasticsearch` storage by default a job is created to clean old traces from it, the options for it are listed below so you can configure it to your use case
+
```
Comment (jpkrohling) [inline README.adoc]: Could you add the code block demarcations, like 

```asciidoc
[source,yaml]
----
storage :
  type: ...
```

This way, we should get appropriate syntax highlighting in some of the supported outputs.

Comment (cfontes) [reply README.adoc]: Sure!

File: README.adoc [hunk 2]
```diff
@@ -311,6 +311,21 @@ The self-provision of an Elasticsearch cluster can be disabled by setting the fl
...
+[source,yaml]
+-----
+storage:
+  type: elasticsearch
+  esIndexCleaner:
+    enabled: false                                // turn the job deployment on and off
+    numberOfDays: 7                               // number of days to wait before deleting a record
+    schedule: "55 23 * * *"                       // cron expession for it to run
```
Comment (jpkrohling) [inline README.adoc]: `s/expession/expression`

**Comment by jpkrohling** [comment]
Thanks @cfontes! This will be merged as soon as Travis gives its green check mark.

**Pull request merged**